app/application.py

```diff
@@ -72,19 +89,12 @@ def logged_in():
             # userID = decodedJWT['username']
             # userName = userID.split('@')[0]
 
-            # Build the survey data (To Do survey type)
-            dataArray = build_survey_data()
-
-            # Filter the data array to remove surveys that shouldn't appear on the To Do page
-            allowedStatuses = ['Not started', 'In progress']
-
-            # TODO - the line below can be commented out to demonstrate sorting. This comment can be removed eventually.
-            dataArray = filter_surveys(dataArray, allowedStatuses)
+            # Filters the data array to remove surveys that shouldn't appear on the To Do page
+            statusFilter = {'status_filter': '["Not started", "In progress"]'}
 
-            # Sort the data array so that the closed Submit by dates appear at the top of the list
-            dataArray = sort_survey_data(dataArray)
+            # Get the survey data (To Do survey type)
+            dataArray = build_survey_data(statusFilter)
```

Comment:
Can the build_survey return any type of error? if it can then it needs to be captured and delt with.